### DISCUSSION TITLE
Seek for help for Educational iOS App Development

### DISCUSSION THREAD

**Discussion by Xshen1991** [community/community#108426] [Discussions]
### Select Topic Area

Question

### Body

Hey everyone,
I am planning to develop an educational software for iOS aimed at helping art students understand two main categories of problems. The first category involves human anatomy, where I aim to utilize image capture and software analysis to render photographed subjects into 3D models, capable of identifying joints and movements. like this>
<img width="100" alt="First" src="https://github.com/community/community/assets/159731043/a0ad4962-90d5-4c3b-9200-c74914fe0a11">
The second category pertains to still-life objects, such as teapots and fruits, and the variations in their shadows under different lighting angles. like this:
<img width="221" alt="second" src="https://github.com/community/community/assets/159731043/e9f3973f-ab69-44db-afc2-bfd876429eee">
I am reaching out here seeking assistance to understand the approximate timeframe and budget required for developing this software. While I am not clear on the specifics of the software's functionality and budget, I believe collaborating with experienced business partners would yield better results. If you are an experienced developer or have relevant advice and resources, I would greatly appreciate your input.

Thanks a ton!
Luna

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Xshen1991 ! The Discussions category is solely related to conversations around the GitHub product Discussions.

We currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
